x-pack/plugins/reporting/server/export_types/png_v2/png_v2.test.ts

```diff
@@ -41,15 +43,32 @@ beforeEach(async () => {
   content = '';
   stream = { write: jest.fn((chunk) => (content += chunk)) } as unknown as typeof stream;
 
-  const mockReportingConfig = createMockConfigSchema({
+  const configType = createMockConfigSchema({
     encryptionKey: mockEncryptionKey,
     queue: {
       indexInterval: 'daily',
       timeout: Infinity,
     },
   });
 
-  mockReporting = await createMockReportingCore(mockReportingConfig);
+  mockReportingCore = await createMockReportingCore(configType);
+  const context = coreMock.createPluginInitializerContext(configType);
+
+  const mockCoreSetup = coreMock.createSetup();
+  const mockCoreStart = coreMock.createStart();
+
+  mockPngExportType = new PngExportType(mockCoreSetup, configType, mockLogger, context);
+
+  mockPngExportType.setup({
+    basePath: { set: jest.fn() },
+  });
+  mockPngExportType.start({
+    savedObjects: mockCoreStart.savedObjects,
+    uiSettings: mockCoreStart.uiSettings,
+    screenshotting: {} as unknown as ScreenshottingStart,
+    esClient: elasticsearchServiceMock.createClusterClient(),
+    reporting: mockReportingCore.getContract(),
+  });
```

Comment:
nit: maybe preferable to keep these lines together:
```suggestion
  mockPngExportType = new PngExportType(mockCoreSetup, configType, mockLogger, context);
  mockPngExportType.setup({
    basePath: { set: jest.fn() },
  });
  mockPngExportType.start({
    savedObjects: mockCoreStart.savedObjects,
    uiSettings: mockCoreStart.uiSettings,
    screenshotting: {} as unknown as ScreenshottingStart,
    esClient: elasticsearchServiceMock.createClusterClient(),
    reporting: mockReportingCore.getContract(),
  });
```